sdk/storage/azure-storage-file-datalake/setup.py

```diff
@@ -72,7 +72,7 @@
     author_email='ascl@microsoft.com',
     url='https://github.com/Azure/azure-sdk-for-python',
     classifiers=[
-        "Development Status :: 5 - Production/Stable",
+        "Development Status :: 4 - Beta",
```

Comment:
Is this a beta/preview? The version change above is stable, and the only change in the changelog is the block size.

Reply:
I want to make the version to 12.1.0b (commented above) let me know if you have any concern!!